controllers/chat/ChatController.js

```diff
@@ -266,15 +267,14 @@ module.exports = {
       const channel = client.channel('messaging', {members});
       const createdChannel = await channel.create();
 
-      await channel.updatePartial({
-        set: {
+      const {betterChannelMember, betterChannelMemberObject} =
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
           channel_type: CHANNEL_TYPE.ANONYMOUS,
           anon_user_info_color_code,
           anon_user_info_color_name,
           anon_user_info_emoji_code,
           anon_user_info_emoji_name
-        }
-      });
+        });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of `channel.updatePartial` with `BetterSocialCore.chat.updateBetterChannelMembers` appears to be a positive change. It's important to ensure that `updateBetterChannelMembers` function handles all the necessary updates and error handling that `updatePartial` was doing. If not, there might be missing functionality or unhandled errors.
```diff
-      await channel.updatePartial({
-        set: {
-           channel_type: CHANNEL_TYPE.ANONYMOUS,
-           anon_user_info_color_code,
-           anon_user_info_color_name,
-           anon_user_info_emoji_code,
-           anon_user_info_emoji_name
-        }
-      });
+      const {betterChannelMember, betterChannelMemberObject} =
+        await BetterSocialCore.chat.updateBetterChannelMembers(channel, createdChannel, true, {
+           channel_type: CHANNEL_TYPE.ANONYMOUS,
+           anon_user_info_color_code,
+           anon_user_info_color_name,
+           anon_user_info_emoji_code,
+           anon_user_info_emoji_name
+        });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->